README.md

```diff
@@ -319,6 +319,51 @@ warnings = validate.validate_pantry(pantry)
 pantry.validate()
 ```
 
+### Using A Mongo DB
+
+There are a couple of things to add when using mongo with weave. 
+You need to give a Pantry object a mongo client.
+
```

Comment:
Reword "You need to give a Pantry ob..." to "Provide a pymongo client when intializing a pantry object"